python/torch_mlir_e2e_test/test_suite/reshape_like.py

```diff
@@ -429,3 +429,40 @@ def forward(self, a):
 def ReshapeAliasCollapseModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(2, 4))
 
+# ==============================================================================
+
+class ReshapeSameRankModule(torch.nn.Module):
```

Comment:
nit: can you move these two tests up so that they are next to the other `Reshape` tests?